WhatsApp.user.css

```diff
@@ -239,7 +239,7 @@
     --mgreen: /*[[mgreen]]*/;
     --mblue: /*[[mblue]]*/;
     --hover-delay: /*[[hoverDelay]]*/;
-    --image: url(https://media.discordapp.net/attachments/310799529161129984/417408747812552715/zilesanamin.jpg);
+    --image: url(images/zilesanamin.jpg);
```

Comment:
This will try to load the image from `https://web.whatsapp.com/images/zilesanamin.jpg` and fail.

Replace it with this: `url("https://raw.githubusercontent.com/vednoc/onyx/master/images/zilesanamin.jpg")`

Reply:
Oops, My Bad. Totally forgot about that. 😅